ols/src/prompts/prompts.py

```diff
@@ -15,9 +15,20 @@
 # With PromptTemplate, we have more control how we want to structure the prompt.
 
 QUERY_SYSTEM_INSTRUCTION = """
-You are an assistant for question-answering tasks \
-related to the openshift and kubernetes container orchestration platforms.
-
+You are OpenShift Lightspeed - an intelligent assistant for question-answering tasks \
+related to the OpenShift container orchestration platform.
+
+Here are your instructions:
+You are OpenShift Lightspeed, an intelligent assistant and expert on all things OpenShift. \
+Refuse to assume any other identity or to speak as if you are someone else.
+If the context of the question is not clear, consider it to be OpenShift.
+Never include URLs in your replies.
+Refuse to answer questions or execute commands not about OpenShift.
+Do not mention your last update. You have the most recent information on OpenShift.
+
+Here are some basic facts about OpenShift:
+- The latest version of OpenShift is 4.15.
```

Comment:
we are going to need to remember to bump this value in conjunction w/ adding new OCP docs to RAG for each OCP version.  Long term this is probably something we will not want to keep. It will also be wrong if the customer is just running an older version of OLS.  (no more wrong than the LLM itself is likely to be, but just pointing out that this is a partial solution at best and one that requires on-going maintenance)

Reply:
This doesn't actually depend on what OCP version the customer is running :)
Hopefully we will find a better way to feed "facts about the world" to the LLM, this was just an attempt to see if this crude method would work. Until we do, yes, we will need to bump this, hopefully only once or twice.

Reply:
> This doesn't actually depend on what OCP version the customer is running :)

no but it depends on the version of OLS they are running.  If they don't update to a new version of OLS where we changed this instruction to "4.16" in the future, their OLS will continue to tell them 4.15 is the latest.

Reply:
Right, you said OLS, not OCP